patches/chrome-android-java-src-org-chromium-chrome-browser-toolbar-bottom-BottomToolbarVariationManager.java.patch

```diff
@@ -0,0 +1,67 @@
+diff --git a/chrome/android/java/src/org/chromium/chrome/browser/toolbar/bottom/BottomToolbarVariationManager.java b/chrome/android/java/src/org/chromium/chrome/browser/toolbar/bottom/BottomToolbarVariationManager.java
+index 5aca8d3e33981d352282b868dd2ffaaa32a45a74..278dd69cb5a0bcd7deb23e9fef68ff1ddc062b4c 100644
+--- a/chrome/android/java/src/org/chromium/chrome/browser/toolbar/bottom/BottomToolbarVariationManager.java
++++ b/chrome/android/java/src/org/chromium/chrome/browser/toolbar/bottom/BottomToolbarVariationManager.java
+@@ -41,9 +41,9 @@ public class BottomToolbarVariationManager {
+             return Variations.HOME_SEARCH_TAB_SWITCHER;
+         }
+         sVariation = FeatureUtilities.getBottomToolbarVariation();
+-        if (sVariation.equals(Variations.NONE)) {
++/*        if (sVariation.equals(Variations.NONE)) {
+             return Variations.HOME_SEARCH_TAB_SWITCHER;
+-        }
++        }*/
+         return sVariation;
+     }
+ 
+@@ -52,7 +52,7 @@ public class BottomToolbarVariationManager {
+      *         in the current variation.
+      */
+     public static boolean isShareButtonOnBottom() {
+-        return FeatureUtilities.isBottomToolbarEnabled()
++        return FeatureUtilities.isBottomToolbarEnabled() && !getVariation().equals(Variations.NONE)
```

Comment:
same as above, this can be change to `&& !getVariation().equals(Variations.NONE)` which would actually make it more consistent with the rest of the code anyway